src/interactions/music/next.ts

```diff
@@ -0,0 +1,34 @@
+import { CommandInteraction } from 'discord.js';
+import { useQueue } from 'discord-player';
+import Interaction from '../../models/Interaction';
+
+export default class Next extends Interaction<CommandInteraction> {
+    name = 'next';
+
+    description = 'Change to the next track';
+
+    options = [];
+
+    // eslint-disable-next-line class-methods-use-this
+    execute = async (interaction: CommandInteraction) => {
+        const queue = useQueue(interaction.guild!);
+        const queueLength = queue?.getSize() || 0;
+
```

Comment:
Consider adding a check to ensure that the caller is actually in a voice channel. I don't think you should be allowed to skip tracks if you're not in voice